app/stores/userEventStore.ts

```diff
@@ -0,0 +1,31 @@
+import type { Event } from "nostr-tools";
+import { create } from "zustand";
+import { createJSONStorage, devtools, persist } from "zustand/middleware";
+
+interface UserEventStore {
+  userEvents: Record<string, Array<Event>>;
```

Comment:
we can keep the user bounty events in the main bounty event store

Reply:
Done - yep i went back and forth on that. Makes more sense to have them in the same object.